### PR TITLE
sigil: update to 1.4.2

### PR DESCRIPTION
**Pull request by ndowens**

### CODE REVIEW

File: srcpkgs/sigil/template [hunk 1]
```diff
@@ -19,7 +19,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/Sigil-Ebook/Sigil"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=8e4919bde7b0e530e12a6feb8956e92e662270187d924a0102e505e942acacb1
+checksum=e31b7c84507ffe9ca896d8a169867923d3274c02d957bd59e5c6b1fd12b5e4c8
 python_version=3
 
 case "$XBPS_TARGET_MACHINE" in
```
Comment (Piraty) [inline srcpkgs/sigil/template:25]: please align with what qt5-webengine's template actually does to limit building on arm

**Review by Piraty — CHANGES_REQUESTED** [review]

File: srcpkgs/sigil/template (hunk 1 above)
Comment (ndowens) [reply srcpkgs/sigil/template]: Updated, is the change ok?

Comment (ericonr) [reply srcpkgs/sigil/template]: Shouldn't it fail to build automatically once `qt5-webengine-devel` can't be found as a `makedepends`?

Comment (ndowens) [reply srcpkgs/sigil/template]: @ericonr Correct, it does without adding anything. Normal xbps behavior

Comment (ericonr) [reply srcpkgs/sigil/template]: I don't see why it needs "broken", then.

Comment (ndowens) [reply srcpkgs/sigil/template]: @ericonr Removed broken in this update.

Comment (Piraty) [reply srcpkgs/sigil/template]: @ericonr i think some maintainers like it that way as the builders bail out early and more verbose this way.
Fine with me though

**Pull request merged**